util/blockdevice/util_test.go

```diff
@@ -124,13 +231,554 @@ func TestGetCapacityOrError(t *testing.T) {
 		name := name
 		mock := mock
 		t.Run(name, func(t *testing.T) {
-			_, err := GetCapacityOrError(mock.src)
+			_, err := GetPhysicalSectorSizeOrError(mock.src)
+			if mock.isErr && err == nil {
+				t.Fatalf("Expected error got none")
+			}
+			if !mock.isErr && err != nil {
+				t.Fatalf("Expected no error got [%+v]", err)
+			}
+		})
+	}
+}
+
+func TestGetHostNameOrError(t *testing.T) {
+	var tests = map[string]struct {
+		src    unstructured.Unstructured
+		result string
```

Comment:
can result be renamed to expect or expectHostName

Reply:
expectedHostName